### ISSUE TITLE
YCBA x YUAG shared exhibitions

### ISSUE THREAD

**Issue by edgartdata** [ycba-cia/blacklight-collections2#337]
Hi Yer @yervangcohen, just summarizing here our conversation with Eric @yulgit1 and David @dcp99:

YCBA and YUAG have about 40 exhibitions in common, i.e. exhibitions that included objects from both museums: [YCBA list of YCBA-YUAG exhibitions.xlsx](https://github.com/ycba-cia/blacklight-collections2/files/7721273/YCBA.list.of.YCBA-YUAG.exhibitions.xlsx)

For the benefit of LUX, YCBA is provisioning for recording YUAG's json files in TMS as additional exhibition IDs in the exhibition record text entry field. Does that still jive with your plan @yervangcohen ?

As with the current exhibition ID that we are currently vending to LIDO (616 for example), the information for the new exhibition ID is also coming from the Exhibitions Module. The new information includes 2 elements: the lido:type and the exhibition ID value.

The difference is that the lido:type for the new/additional exhibition ID is “LUX YUAG exhibition” ( and not “local”). “LUX YUAG exhibition” is the TMS exhibition record text entry type (see screenshot below). I guess the lido:type for the new/additional exhibition ID could be hard coded or picked up from the text type. 

The value for the new/additional exhibition ID is in the TMS exhibition record Text Entry text field, in this case 
http://media.art.yale.edu/content/lux/exh/307686.json (but they will change for every exhibition).

![image](https://user-images.githubusercontent.com/4093644/146231205-7e5c6aa3-21eb-4569-97cb-a75bb5076b6b.png)

I have entered http://media.art.yale.edu/content/lux/exh/307686.json as "LUX YUAG exhibition" for YCBA exhibition ID 616 _Responding to Kahn: A Sculptural Conversation_. TMS objectID 6113 is in that exhibition 616 and here is the xml for objectID 6113 for testing https://harvester-bl.britishart.yale.edu/oaicatmuseum/OAIHandler?verb=GetRecord&identifier=oai:tms.ycba.yale.edu:6113&metadataPrefix=lido

<lido:eventSet> 
<lido:displayEvent>Responding to Kahn: A Sculptural Conversation (Yale University Art Gallery, 2006-11-30 - 2007-07-08)</lido:displayEvent>   
    <lido:event>
        <lido:eventID lido:type="local">616</lido:eventID> 
        **<lido:eventID lido:type="LUX YUAG exhibition">http://media.art.yale.edu/content/lux/exh/307686.json</lido:eventID>** 
        <lido:eventType>
            <lido:conceptID lido:type="local">300054766</lido:conceptID>
            <lido:term>exhibition</lido:term>
        </lido:eventType>
        <lido:eventName>
            <lido:appellationValue>Responding to Kahn: A Sculptural Conversation</lido:appellationValue>
   <lido:sourceAppellation>Yale Center for British Art</lido:sourceAppellation>
        </lido:eventName>

**Comment by yulgit1** [comment]
One bit of work with this will be supplying to 40 LUX URLs, like for YCBA for YUAG would be something like:

https://ycba-lux.s3.amazonaws.com/v3/activity/00/000e7f5e-d6bf-4439-a5ed-0290479cb3b6.json

And vice-versa YUAG to YCBA like:

http://media.art.yale.edu/content/lux/exh/307686.json

Each of the 40 will have to be manually correlated with the appropriate exhibit and entered into TMS.

**Comment by edgartdata** [comment]
@yervangcohen @yulgit1 Going back to this issue to check how YUAG YCBA shared exhibitions are coming through LUX. 

I don't see "Colorful Impressions" as a stand alone event in LUX today 2/10/2023: https://lux-front-prd.collections.yale.edu/view/results/objects?q=%22Colorful%20Impressions%22&page=1

It is correctly mentioned in this YCBA object: https://lux-front-prd.collections.yale.edu/view/object/c6f80039-980e-4351-b48b-bc137c5338e4 but clicking on "exhibit set" for it (In the Where is it at Yale?) only brings up YCBA objects. @yulgit1 I added YUAG's json for it in TMS a while back https://media.art.yale.edu/content/lux/exb/2482.json  Didn't we want to vend these to LUX?

Here is the list of approved shared exhibits:
[YCBA list of YCBA-YUAG exhibitions with json for LUX.xlsx](https://github.com/ycba-cia/blacklight-collections2/files/10708918/YCBA.list.of.YCBA-YUAG.exhibitions.with.json.for.LUX.xlsx)

**Comment by yulgit1** [comment]
@edgartdata I don't follow.

I see this query returning the Colorful Impressions exhibition event:

query: https://lux-front-prd.collections.yale.edu/view/results/events?q=%22Colorful%20Impressions%22&page=1
event: https://lux-front-prd.collections.yale.edu/view/activity/ef9bfa8b-1d19-4b44-a004-a5ab75cdf6d7

I see this above exhibition event with contributing records from both YCBA and YUAG, where YUAG has equivalent to YCBA and YCBA has equivalent to YUAG (that https://media.art.yale.edu/content/lux/exb/2482.json).  Thus they are merged.

I see this above exhibition event including 2 sets, one with the YCBA objets and one with the YUAG objects.  

I see the van Dyck object with links to the YCBA set.  Perhaps what you're getting at is that while there's a link from the object to the YCBA set (which of course won't have YUAG objects), there is no link from the object to the merged exhibition?  Maybe LUX should show that?

**Comment by yulgit1** [comment]
No, wait, I take that back, the part about "I see this above exhibition event including 2 sets, one with the YCBA objets and one with the YUAG objects."  The exhibition in LUX does not show the YCBA set despite the YCBA having the contributing record with:

```
"used_specific_object": [
    {
      "id": "https://ycba-lux.s3.amazonaws.com/v3/set/da/da24b38c-d706-4fc9-872d-38d46899dc1c.json",
      "type": "Set",
      "_label": "Exhibit set for \"Colorful Impressions - The Printmaking Revolution in Eighteenth-Century France (Yale University Art Gallery, 2008-01-29 - 2008-05-04)\""
    }
  ],
```

It should show that set, given that right?

**Comment by edgartdata** [comment]
Correct: the object results for the search "Colorful Impressions" does not currently return YCBA objects, only the 6 YUAG prints, and the 3 catalogs for the exhibition. 

The YCBA objects are in the [Event record for the exhibition](https://lux-front-prd.collections.yale.edu/view/activity/ef9bfa8b-1d19-4b44-a004-a5ab75cdf6d7) (although it is labeled as event type = exhibition catalog, which aren't events, exhibitions are ... but that's a separate issue) but they are not coming up in the Objects tab when doing a search for "Colorful Impressions".
Should we add a new lux github issue?

**Comment by yulgit1** [comment]
Yes.

To make sure I have it straight. (1) Objects are in a set for an exhibition (good). (2) But search doesn't find those objects based on set name, (3) in at least one case (Color Impressions) that set is missing from the exhibition despite having a "used_specific_object" of that set in the contributing record. (4) objects are linked to sets not exhibitions, (it is mostly 1:1 between set and exhibition, but not the case with shared exhibitions) (5) no link from exhibition set to exhibition.

And another now in thinking about this: Objects can be part of exhibition sets, but then how do you find a set, including in particular an exhibition set from a search (ie. which tab would it go under)?

**Comment by edgartdata** [comment]
Yes to all 5 points and yes also to your question about finding sets.
